internal/factory/git_projection_test.go

```
@@ -0,0 +1,110 @@
+			t.Fatalf("refreshed projection contains forbidden %q: %v", forbidden, err)
+		}
+	}
+	repeatedProjection, err := prepareGitMetadataProjection("run-refresh", repositoryPath, worktreePath)
+	if err != nil {
+		t.Fatalf("repeat Git metadata projection refresh: %v", err)
+	}
+	if repeatedProjection != projection {
+		t.Fatalf("repeated projection = %q, want stable path %q", repeatedProjection, projection)
+	}
+
+	command := exec.Command("git", "diff", "--no-ext-diff", "--binary", "--unified=80", baseSHA, checkpointSHA, "--", ".")
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)